src/components/ui-shell/side-nav-footer.hbs

```diff
@@ -1,14 +1,14 @@
 <footer class="{{@root.prefix}}--side-nav__footer">
-  <button
-    class="{{@root.prefix}}--side-nav__toggle"
-    role="button"
-    title="Close the side navigation menu">
+  <button class="{{@root.prefix}}--side-nav__toggle" role="button" title="Close the side navigation menu">
     <div class="{{@root.prefix}}--side-nav__icon">
-      {{#if expanded}}
-      <svg aria-hidden="true"width="20" height="20" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><path d="M17.414 16L24 9.414 22.586 8 16 14.586 9.414 8 8 9.414 14.586 16 8 22.586 9.414 24 16 17.414 22.586 24 24 22.586 17.414 16z"/></svg>
-      {{else}}
-      <svg aria-hidden="true"width="20" height="20" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><path d="M22 16L12 26l-1.414-1.414L19.172 16l-8.586-8.586L12 6l10 10z"/></svg>
-      {{/if}}
+      <svg class="{{@root.prefix}}--side-nav__icon--collapse" aria-hidden="true" width="20" height="20" xmlns="http://www.w3.org/2000/svg"
```

Comment:
Should we use the `carbon-icon` helper for these 20x20 icons?

Reply:
yep, edited!